packages/syft/src/syft/core/node/worker.py

```diff
@@ -7,13 +7,20 @@
 from functools import partial
 import hashlib
 import os
+import sys
 from typing import Any
 from typing import Callable
 from typing import List
 from typing import Optional
 from typing import Type
 from typing import Union
 
+if sys.version_info >= (3, 11):
```

Comment:
we cant support python 3.11 yet but even if we can, typing_extensions defaults to native implementations where available:
<img width="218" alt="Screenshot 2023-04-05 at 3 52 08 pm" src="https://user-images.githubusercontent.com/2882739/229992673-303aa8a6-8813-4a95-9c5c-cb767b32a58a.png">

https://github.com/python/typing_extensions/blob/37909ec67adf56070fa7218efa475e87812834e5/src/typing_extensions.py#L1658

Reply:
Yup that's always the case with `typing_extensions`. I agree that leaving it as just `from typing_extensions import ...` no matter the Python version would be no difference technically. The benefit is that it reminds us to move it to `from typing import ...` later on when the minimum Python version supported becomes 3.11. I agree the benefit for doing that in this case is extremely small, more about just following a recommended pattern.